app/views/podcast_planner/_form_side.html.erb

```diff
@@ -1,56 +1,53 @@
-<div class="card">
-  <div class="card-header-primary">
-    <h5 class="card-title"><%= t(".header.preview") %></h5>
+<%= form_with(
+      url: podcast_planner_path(@podcast),
+      method: :post
+    ) do |form| %>
+  <div class="mb-4">
+    <h3 class="mb-3"><%= t(".header.publish_time") %></h3>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.select :publish_time, time_options %>
+        <%= form.label :publish_time %>
+      </div>
+    </div>
   </div>
 
-  <%= form_with(
-        url: podcast_planner_path(@podcast),
-        method: :post
-      ) do |form| %>
-      <div class="card-body">
-        <div class="row mb-4">
-          <div class="col-lg-6">
-            <h3 class="fw-bold"><%= t(".header.publish_time") %></h3>
-            <div class="form-floating">
-              <%= form.select :publish_time, time_options %>
-              <%= form.label :publish_time %>
-            </div>
-          </div>
+  <div class="mb-4">
+    <h3 class="mb-3"><%= t(".header.audio_segments") %></h3>
+    <div class="col-lg-12">
+      <div class="form-floating">
+        <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2} %>
+        <%= form.label :segment_count %>
+      </div>
+    </div>
+  </div>
 
-          <div class="col-lg-6">
-            <h3 class="fw-bold"><%= t(".header.audio_segments") %></h3>
-            <div class="form-floating">
-              <%= form.number_field :segment_count, {min: 1, max: 10, step: 1, value: 2} %>
-              <%= form.label :segment_count %>
-            </div>
+  <div class="card p-4 mt-4mb-4">
+    <%= turbo_frame_tag "preview", target: "_top" do %>
+      <div data-controller="pager count" data-pager-range-value="6">
+        <div class="d-flex justify-content-between">
+          <h3 class="mb-3"><%= t(".header.number_of_drafts") %>
+            <span data-count-target="display"></span>
```

Comment:
new span here. the innerHTML of this will update with the number of selected items, as determined by the new `count_controller`.

Reply:
I would probably render a number in here to start?  So it has the correct value in HTML, before javascript even takes a pass at it.